### PR TITLE
doc: add development & testing instruction in CONTRIBUTING.md

### LINKED ISSUE #79: CONTRIBUTING.md: Helm Chart testing instruction and scripts for constributors
using minikube and manifest for volume

### PR DESCRIPTION
**Pull request by hiteshnayak305**
### Description of the change

Add documentation for testing helm chart locally by deploying in minikube.

### Applicable issues

  - fixes #79

### CODE REVIEW

File: charts/immich/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 apiVersion: v2
 description: A chart to power Immich (immich.app) running on kubernetes
 name: immich
-version: 0.6.0
+version: 0.6.1
```
Comment (bo0tzz) [inline charts/immich/Chart.yaml:5]: This shouldn't need a version bump

**Review by bo0tzz — CHANGES_REQUESTED** [review]
Thanks!

File: charts/immich/Chart.yaml (hunk 1 above)
Comment (hiteshnayak305) [reply charts/immich/Chart.yaml:5]: chart immutability will break no ? we might need to change that section as well in CONTRIBUTING.md.

Comment (bo0tzz) [reply charts/immich/Chart.yaml:5]: If the version doesn't change then the chart release doesn't run at all

**Review by bo0tzz — APPROVED** [review]
Ty!

**Pull request merged**